src/main/java/com/google/api/codegen/PageStreamingConfig.java

```diff
@@ -94,7 +94,7 @@ public static PageStreamingConfig createPageStreaming(
               resourcesFieldName));
     }
 
-    if (requestTokenField == null || responseTokenField == null || resourcesField == null) {
+    if (requestTokenField == null && responseTokenField == null && resourcesField == null) {
```

Comment:
Why does gRPC streaming affect PageStreamingConfig? And what does this change mean? For regular page streaming, all three of these fields must not be null.